src/core/image/handlers/thumbnail.handler.ts

```diff
@@ -0,0 +1,55 @@
+import { ImageService } from '../image.service';
```

Comment:
Isn't it the same as resize?
As I remember we decided that thumbnail is not necessary...?